project/roadmap.md

```diff
@@ -59,8 +59,9 @@ A tick indicates an item is implemented and has automated tests in place.
 - [ ]	PUT/_api/simple/all-keys Read all documents
 
 #### Graph API
+<<<<<<< HEAD
```

Comment:
This extra line slipped in after resolving conflicts. Let's remove it and it's ready to merge.